MaterialLib/MPL/Properties/CreatePermeabilityMohrCoulombFailureIndexModel.cpp

```diff
@@ -0,0 +1,83 @@
+/**
+ * \file
+ * \copyright
+ * Copyright (c) 2012-2020, OpenGeoSys Community (http://www.opengeosys.org)
+ *            Distributed under a Modified BSD License.
+ *              See accompanying file LICENSE.txt or
+ *              http://www.opengeosys.org/project/license
+ *
+ * Created on June 5, 2020, 9:07 AM
+ */
+
+#include "CreatePermeabilityMohrCoulombFailureIndexModel.h"
+
+#include <string>
+
+#include "BaseLib/ConfigTree.h"
+#include "Parameter.h"
+#include "ParameterLib/CoordinateSystem.h"
+#include "ParameterLib/Parameter.h"
+#include "ParameterLib/Utils.h"
+#include "PermeabilityMohrCoulombFailureIndexModel.h"
+
+namespace MaterialPropertyLib
+{
+std::unique_ptr<Property> createPermeabilityMohrCoulombFailureIndexModel(
+    int const geometry_dimension,
+    BaseLib::ConfigTree const& config,
+    std::vector<std::unique_ptr<ParameterLib::ParameterBase>> const& parameters,
+    ParameterLib::CoordinateSystem const* const local_coordinate_system)
+{
+    if ((geometry_dimension != 2) && (geometry_dimension != 3))
+    {
+        OGS_FATAL(
+            "The PermeabilityMohrCoulombFailureIndexModel is implemented only "
+            "for 2D or 3D problems");
+    }
+
+    //! \ogs_file_param{properties__property__type}
+    config.checkConfigParameter("type",
+                                "PermeabilityMohrCoulombFailureIndexModel");
+
+    // Second access for storage.
+    //! \ogs_file_param{properties__property__name}
+    auto property_name = config.peekConfigParameter<std::string>("name");
+
+    DBUG("Create PermeabilityMohrCoulombFailureIndexModel property {:s}.",
+         property_name);
+
+    std::string const& parameter_name =
+        //! \ogs_file_param{properties__property__PermeabilityMohrCoulombFailureIndexModel__initial_permeability}
+        config.getConfigParameter<std::string>(
+            "initial_permeability_parameter");
```

Comment:
The ogs_file_param string does not match the parsed tag.
```suggestion
        //! \ogs_file_param{properties__property__PermeabilityMohrCoulombFailureIndexModel__initial_permeability}
        config.getConfigParameter<std::string>(
            "initial_permeability");
```

Reply:
Changed.